packages/gatsby-source-wordpress/README.md

```diff
@@ -135,6 +135,9 @@ module.exports = {
         ],
         // Blacklisted routes using glob patterns
         excludedRoutes: ["**/posts/1456"],
+        // Set whether or not thumbnail URLS are sourced.
+        // Defaults to false.
+        sourceThumbnails: false,
```

Comment:
Let's call this `keepMediaDetails` to make obvious what this option does

It isn't immediately obvious what `sourceThumbnails` does without enough context

Reply:
Just saw this. Did you make the changes or should i?

Reply:
I haven't made them yet but I can if you like!